adapters/sharethrough/sharethrough_test.go

```diff
@@ -104,7 +106,9 @@ func TestFailureMakeRequests(t *testing.T) {
 	}{
 		"Returns nil if failed to generate request": {
 			input: &openrtb.BidRequest{
-				App: &openrtb.App{Ext: []byte(`{}`)},
+				Site: &openrtb.Site{
+					Page: "test.com",
+				},
```

Comment:
Both here and above at line 60, a request with App was replaced by a request with Site.  Does this mean App is no longer supported and therefore no App test is needed?

Reply:
App is not supported (actually it was never supported, the App in the test was a mistake), I improved the error message to make it more explicit